code/modules/multiz/structures.dm

```diff
@@ -108,6 +107,35 @@
 /obj/structure/multiz/ladder/attack_generic(var/mob/M)
 	attack_hand(M)
 
+/obj/structure/multiz/ladder/proc/throw_through(var/obj/item/C, var/mob/throw_man)
+	if(istype(throw_man,/mob/living/carbon/human))
+		var/mob/living/carbon/human/user = throw_man
+		if(istype(C,/obj/item/weapon/grenade))
```

Comment:
Don't see why this should only work with nades.

Reply:
It would be nice to let players throw down other stuff like guns or money, etc.

Reply:
done